src/validators/account-validator.js

```diff
@@ -0,0 +1,231 @@
+const  {check, param}  =  require ( 'express-validator' );
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const  {check, param}  =  require ( "express-validator" );
```